cmd/goquery_completion/conditional.go

```diff
@@ -66,14 +67,15 @@ func nextAll(prevprev, prev string, openParens int) []suggestion {
 			s("host", false),
 			s("net", false),
 			s("dport", false),
+			s("port", false),
 			s("proto", false),
 		}
 	case "dip", "sip", "dnet", "snet", "dst", "src", "host", "net":
 		return []suggestion{
 			s("=", false),
 			s("!=", false),
 		}
-	case "dport", "proto":
+	case "dport", "port", "proto":
```

Comment:
FYI: added this alias because I forget the `d` in `dport` half the time ;)

Reply:
Makes sense (it's a bit ambiguous of course by definition, but since we don't _have_ source ports in goDB I think there's no issue with expectation management)...